apps/expo/src/temp-data/top-trekers/topTrekersList.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Can you find a way to make it typesafe without disabling eslint?

Reply:
Ok lang. This data is temporary to display the component. Will change once user schema is implemented sa db